# Copyright (c) 2024-2025 Voyant Photonics, Inc.
#
# This example code is licensed under the MIT License.
# See the LICENSE file in the repository root for full license text.

# These parameters are used in the MCAP -> voyant .bin conversion

mcap_input: '' # Path to your bin file
bin_output: '' # Desired output path
